internal/attestation/crafter/crafter.go

```diff
@@ -43,6 +43,28 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+// attestationCraftingMaterial is a map of material types to their corresponding schemaapi.CraftingSchema_Material_MaterialType
+var attestationCraftingMaterial = func() map[string]schemaapi.CraftingSchema_Material_MaterialType {
+	res := make(map[string]schemaapi.CraftingSchema_Material_MaterialType)
+	for k, v := range schemaapi.CraftingSchema_Material_MaterialType_value {
+		if k != "MATERIAL_TYPE_UNSPECIFIED" {
+			res[strings.Replace(k, "MATERIAL_TYPE_", "", 1)] = schemaapi.CraftingSchema_Material_MaterialType(v)
+		}
+	}
+	return res
+}
+
+// ListAvailableMaterialKind returns a list of available material kinds
+func ListAvailableMaterialKind() []string {
```

Comment:
btw, this method could be added to the crafting schema package. 

So you could do `schemaapi.ListAvailableMaterialKinds()`